hwdb/60-evdev.hwdb

```diff
@@ -45,6 +45,7 @@
 
 #  Macbook2,1 (late 2006), single-button touchpad
 evdev:input:b0003v05ACp021B*
+
```

Comment:
These empty lines break the format, as all consecutive "evdev:" lines apply to the same indented block of `EVDEV_*` properties set below. Please remove them again.